tests/utils.py

```diff
@@ -49,3 +51,99 @@
     '_id': 'yes! yes! yes!',
     'count': 0
 }
+
+
+TEST_SCHEMA = update_schema(BASEXMLSCHEMA, {
+    "properties": {
+        "title1": "//dc:title/node()",
+        "title2": ["//dc:title/node()", lambda x: x.lower()],
+        "title3": ["//dc:title/node()", "//dc:title/node()", lambda x, y: x + y.lower()]
```

Comment:
I added them in, but I'm still not sure what I think about them. Perhaps we, @chrisseto and @erinspace could chat about it tomorrow. I think the ```((args, kwargs), callable)``` pattern doesn't work well from a usability perspective (after having written the test cases for it)